src/Client.php

```diff
@@ -247,6 +247,43 @@ public function dropDatabase(string $databaseName, array $options = [])
         return $operation->execute($server);
     }
 
+    /**
+     * Returns a collection instance.
+     *
+     * If the collection does not exist in the database, it is not created when
+     * invoking this method.
+     *
+     * @see Collection::__construct() for supported options
+     * @param string $databaseName   Name of the database containing the collection
+     * @param string $collectionName Name of the collection to select
+     * @param array  $options        Collection constructor options
+     * @throws InvalidArgumentException for parameter/option parsing errors
```

Comment:
I suppose you want the doc block for the note about "does not exist", but the parameter annotations here have no value. The code is self-documenting.

Reply:
Does it helps static analysis to tell which kind of exception can be thrown?

Reply:
I left `@throws` in for the time being, since they are included in all other methods as well. I'm on the fence about the usefulness of them, I can see them being useful when coupled with static analysis, but since we don't have anything to enforce correctness of `@throws` statements, we may reconsider having them for methods that don't directly throw any exceptions.